.gitignore

```diff
@@ -1,7 +0,0 @@
-.idea/*
-.gradle/*
-build/*
-enigma-backup/*
-app/release/*
-app/debug/*
-app/build/*
```

Comment:
Please don't delete this file

Reply:
This is also my mistake, same as for the service. My apologies.